### PR TITLE
Add Scadutree Fragments and Revered Spirit Ashes from Gravesite Plain

### PR DESCRIPTION
**Pull request by DanyBoyP123**
The guide for Scadutree Fragments on the wiki isn't organized by regions, making it harder to just copy/paste without checking the game. I used another guide and I wanted your opinion about it before doing the rest.

### CODE REVIEW

File: data/lists/upgrades.ts [hunk 1]
```diff
@@ -360,9 +360,76 @@ export const UpgradesList: Array<ListType> = [
   },
   {
     id: "e3229bbc-73dc-41c4-b2b5-15f079a7de8d",
-    name: "DLC Area",
```
Comment (Gobluebro) [inline data/lists/upgrades.ts:363]: Jesus did I make these live by mistake? 😬

**Comment by Gobluebro** [comment]
> on the wiki isn't organized by regions, making it harder to just copy/paste without checking the game

I see what you mean yeah that is difficult.

My idea would be to take the regions on a  [map like this](https://cdn.mos.cms.futurecdn.net/xpc4jLS4nBcU7KWuUazgXH-970-80.jpg.webp)

And then sort of compare with how the fragments are on the map like here 
https://eldenring.wiki.fextralife.com/Shadow+of+the+Erdtree+Map
image

Then grouping them together would be possibly easier.

> I used another guide
What guide did you use? I'd be interested in seeing how they group them

**Comment by DanyBoyP123** [comment]
I used the guides from rockpapershotgun:
https://www.rockpapershotgun.com/elden-ring-shadow-of-the-erdtree-revered-spirit-ash-locations
https://www.rockpapershotgun.com/elden-ring-shadow-of-the-erdtree-scadutree-fragment-locations

**Comment by Gobluebro** [comment]
> Your idea is probably the best but it will take quite a bit of time. My suggestion would be to complete what I started, since it would take like an hour or two, and do it your way later. If we keep the same ids, it would be backward compatible.

I'm fine with this. They seem to have a good region names for each.

**Comment by DanyBoyP123** [comment]
I've added the rest.

**Review by Gobluebro — APPROVED** [review]
LGTM

**Comment by Gobluebro** [comment]
Thanks for the help!

**Pull request merged**